### ISSUE TITLE
BitBox multiple accounts path error in UTXO vs Account model

### ISSUE THREAD

**Issue by ruigil** [digitalbitbox/bitbox-wallet-app#2125]
I bought a new BitBox02 and tried to restore my multiple Ethereum accounts created with my Ledger, but I could only retrieve the first one.

In Ledger the multiple accounts are created with the path 
m/44'/60'/0'/0/0
m/44'/60'/1'/0/0
m/44'/60'/2'/0/0

But in BitBox they are created with the following path
m/44'/60'/0'/0/0
m/44'/60'/0'/0/1
m/44'/60'/0'/0/2

As I understand the BIP39 standard, the path format is m / purpose' / coin_type' / account' / change / address_index.
The 'address index' is for UTXO models, so It seems that for Ethereum accounts the BitBox path is not correct ?

How can I retrieve my multiple Ethereum accounts with my new BitBox02 ?

**Comment by ruigil** [comment]
[https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#user-content-Path_levels](url)

**Comment by benma** [comment]
Hi. BIP-44 does not directly apply to Ethereum and there is no one standard that everyone follows unfortunately. The BitBox02 follows the default keypath of myetherwallet.com, which is m/44'/60'/0'/0.

The BitBoxApp and the BitBox02 do not support other keypaths for Ethereum. The BitBox02 firmware could be made to accept other keypaths with a firmware update, but then you'd need to use it with MyEtherWallet or another wallet that lets you specify keypaths. 

The easiest solution here is to transfer the funds from Ledger to BitBox02.

**Comment by ruigil** [comment]
Unfortunately this is not an option for me, because several of my Ethereum accounts are tied to specific smart contract for staking and lending., t is not a question of fund transfer, which means that in the current state BitBox02 is unusable for me.
 
Is there a guide to update the firmware to accept other paths ?

**Comment by benma** [comment]
> Is there a guide to update the firmware to accept other paths ?

We first need to implement the firmware change and release the update. After the update it then should work with MyEtherWallet.

> In Ledger the multiple accounts are created with the path
> m/44'/60'/0'/0/0
> m/44'/60'/1'/0/0
> m/44'/60'/2'/0/0

Are you sure it's not `m/44'/60'/0'`, `m/44'/60'/1'`, `m/44'/60'/2'`, etc. (without the `/0/0`)?  Does Ledger Live display the keypaths?

**Comment by ruigil** [comment]
Yes I'm sure, and the proof is that the first generated public address is the same.
When I click to see advanced info about the account, this is what I get for the first account

{
  "index": 0,
  "freshAddressPath": "44'/60'/0'/0/0",
  "id": "js:2:ethereum:0x33e80<redacted>",
  "blockHeight": <redacted>
} 

And this is what I get for the second. The second account public key doesn't match BitBox02 second account public key.
{
  "index": 1,
  "freshAddressPath": "44'/60'/1'/0/0",
  "id": "js:2:ethereum:0x4A3fa<redacted>",
  "blockHeight": <redacted>
}

**Comment by benma** [comment]
The firmware change needed to allow Ledger Live keypaths can be tracked here:

https://github.com/digitalbitbox/bitbox02-firmware/pull/1072

I checked MyEtherWallet and Rabby, and neither seems to offer the option to choose the keypath manually. So an change there will also be needed to be able to access these accounts through MEW or Rabby.

**Comment by benma** [comment]
Opened upstream issue at Rabby for this: https://github.com/RabbyHub/Rabby/issues/1411

**Comment by a729ahagx** [comment]
How about the status of this feature? I'm using an ledger device, but I would like go change to bitbox. Some of my etherum adresses can't be moved, because of some smart contracts I'm using can't be shifted.

I would highly appreciate adding the ledger live deviation path into bitbox firmware!

**Comment by benma** [comment]
@a729ahagx there has been no progress here due to a lack of demand basically. It seems most Ledger users that switch to BitBox don't experience this or they simply move the funds. You can can consider moving the funds as a workaround.

cc @jadzeidan

**Comment by a729ahagx** [comment]
For just hodl wallets, I agree, moving funds is the way to go. But I have several wallets which are used at lending, borrowing or liquidity mining smart contract. Moving this positions would would have far more financial impact than only the gas fees.
